### ISSUE TITLE
Documentation request: --javascript-delay versus --window-status, who wins?

### ISSUE THREAD

**Issue by kengruven** [wkhtmltopdf/wkhtmltopdf#2616]
In [the documentation][1], it says:

      --javascript-delay <msec>       Wait some milliseconds for javascript
                                      finish (default 200)

and:

      --window-status <windowStatus>  Wait until window.status is equal to this
                                      string before rendering page

But if you specify both, who wins?

I found [a forum posting][2] which says:

> "--javascript-delay 100000 --window-status myrandomstring" 
This will cause wkhtmltopdf to output the pdf as the document looks 
after 100 seconds, or when window.status is set to myrandomstring 
whatever comes first. 

That sounds great, but empirically, it is not what wkhtmltopdf 0.12.2.1 does here.

- If I specify `--window-status` only, it waits a couple seconds until my JS code sets window.status to that value.
- If I specify `--javascript-delay` to 100000, it waits around 100 seconds (or a little more).
- If I specify both, it waits for the longer time.

Either way (max or min), it would be great if the documentation told me what I should expect here.

thanks!

[1]: http://wkhtmltopdf.org/usage/wkhtmltopdf.txt
[2]: https://groups.google.com/forum/#!topic/wkhtmltopdf-general/vWDe5hUAyJ8

**Comment by timenomad** [comment]
I agree, --javascript-delay is not to be confused with timeout functionality.

**Comment by daverogers** [comment]
+1, a `--javascript-timeout` function would be nice. Took me awhile to realize that same forum posting was wrong :P

**Comment by despotgit** [comment]
Neither of the two worked for me. I am on mac. It just stalls if I put one, the other or both, I have the same result, which is a hang. It doesn't work at all.
